varnish/datadog_checks/varnish/varnish.py

```diff
@@ -132,14 +132,17 @@ def check(self, instance):
             varnishadm_path = shlex.split(instance.get('varnishadm'))
             secretfile_path = instance.get('secretfile', '/etc/varnish/secret')
 
+            daemon_host = instance.get('daemon_host', 'localhost')
+            daemon_port = instance.get('daemon_port', '6082')
+
             cmd = []
             if geteuid() != 0:
                 cmd.append('sudo')
 
             if version < LooseVersion('4.1.0'):
                 cmd.extend(varnishadm_path + ['-S', secretfile_path, 'debug.health'])
             else:
-                cmd.extend(varnishadm_path + ['-S', secretfile_path, 'backend.list', '-p'])
+                cmd.extend(varnishadm_path + ['-T', '{}:{}'.format(daemon_host, daemon_port), '-S', secretfile_path, 'backend.list', '-p'])
```

Comment:
I think the condition is broken. Should we check for version 5 explicitly ? Varnish 4.1.9 would endup using the `-T` option.

Reply:
Hey, I added a note to the additional notes section `Tested on version 4.1.1 of Varnish locally and explicitly specifying -T is ok here.` This should be completely OK to do :)